alpha/declcfg/load_test.go

```diff
@@ -961,6 +1166,15 @@ func TestLoadFile(t *testing.T) {
 			expectNumBundles:  6,
 			expectNumOthers:   0,
 		},
+		{
+			name:              "Success/ValidFile/Deprecations",
+			fsys:              validFSD,
+			path:              "deprecations.yaml",
+			assertion:         require.NoError,
+			expectNumPackages: 0,
+			expectNumBundles:  0,
```

Comment:
Wouldn't it be useful to expect deprecations as opposed to not (only) expecting anything else?

Reply:
Had some fun playing with options pattern, reflection and generics here.  In the end, the simplest approach was to inline a function for the comparisons.